aliases.json

```diff
@@ -18,5 +18,5 @@
   "F8131": "dora",
   "F8132": "dora",
   "F8331": "kagura",
-  "F8332": "kagura"
```

Comment:
Sorry to be late for the party, but why the 2nd kagura is deleted now? Below its still kagura": "Sony Xperia XZ (F8331 & F8332)